src/components/expanded-state/UniqueTokenExpandedState.tsx

```diff
@@ -229,7 +229,6 @@ interface UniqueTokenExpandedStateProps {
 const getIsSupportedOnRainbowWeb = (network: Network) => {
   switch (network) {
     case Network.mainnet:
-    case Network.polygon:
```

Comment:
would be nice to add a comment here to re enable when allow list is added to web as well